security/certs.go

```diff
@@ -129,7 +142,7 @@ func RunCreateCACert(certsDir string, keySize int) error {
 // RunCreateNodeCert is the entry-point from the command-line interface
 // to generate node certs and keys:
 // - node.server.{crt,key}: server cert with list of dns/ip addresses
-// - node.{crt,key}: client cert with "node" as the Common Name.
+// - node.client.{crt,key}: client cert with "node" as the Common Name.
```

Comment:
Fair point. I was looking at that the other day for improved client request logging. Is there actually a place at which we can get this information? Seems like the first point of contact we have with a request coming in via the DB gateway is `nodeServer` in `server/node.go`, which already doesn't know who's the peer because `rpc.Server` abstracts all of that away.